neo/Prompt/Commands/Show.py

```diff
@@ -167,3 +173,189 @@ def execute(self, arguments=None):
+        return CommandDesc('notifications', 'show specified contract execution notifications', [p1])
+
+
+class CommandShowAccount(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+        if item is not None:
+            account = Blockchain.Default().GetAccountState(item, print_all_accounts=True)
+
+            if account is not None:
+                print(json.dumps(account.ToJson(), indent=4))
+                return account.ToJson()
+            else:
+                print("Account %s not found" % item)
+                return
+        else:
+            print("Please specify an account address")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('address', 'the address to show')
+        return CommandDesc('account', 'show the assets (NEO/GAS) held by a specified address', [p1])
+
+
+class CommandShowAsset(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+        if item is not None:
+            if item.lower() == "all":
+                assets = Blockchain.Default().ShowAllAssets()
+                print("Assets: %s" % assets)
+                return assets
+
+            if item.lower() == 'neo':
+                assetId = Blockchain.Default().SystemShare().Hash
+            elif item.lower() == 'gas':
+                assetId = Blockchain.Default().SystemCoin().Hash
+            else:
+                try:
+                    assetId = UInt256.ParseString(item)
+                except Exception:
+                    print("Could not find asset from args: %s" % arguments)
+                    return
+
+            asset = Blockchain.Default().GetAssetState(assetId.ToBytes())
+
+            if asset is not None:
+                print(json.dumps(asset.ToJson(), indent=4))
+                return asset.ToJson()
+            else:
+                print("Asset %s not found" % item)
+                return
+        else:
+            print("Please specify an asset hash or name")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('name/assetId/all', 'the name or assetId of the asset, or "all" shows all assets\n\n')
+        f"{' ':>17} Example:\n"
+        f"{' ':>20} 'neo' or 'c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b'\n"
+        f"{' ':>20} 'gas' or '602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7'\n"
+        return CommandDesc('asset', 'show a specified asset', [p1])
+
+
+class CommandShowContract(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+        if item is not None:
+            if item.lower() == "all":
+                contracts = Blockchain.Default().ShowAllContracts()
+                print("Contracts: %s" % contracts)
```

Comment:
I think we can give a better user experience than this
```
neo> show asset all
Assets: dict_keys([b'602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7', b'c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b'])
```
I think we could at least give them a column like looking output showing `asset name` - `asset id`

Reply:
Would you also like this change applied to `ShowAllContracts`? (I formatted `ShowAllAssets` to match `ShowAllContracts`)